resources/dtr_bundle_patient_pat015.json

```
@@ -1374,7 +1363,1506 @@
+            "resourceType": "Patient",
+            "id": "patient",
+            "meta": {
+              "profile": [
+                "http://hl7.org/fhir/us/davinci-cdex/StructureDefinition/cdex-patient-demographics"
+              ]
+            },
+            "identifier": [
+              {
+                "use": "usual",
+                "type": {
+                  "coding": [
+                    {
+                      "system": "http://hl7.org/fhir/us/davinci-hrex/CodeSystem/hrex-temp",
+                      "code": "UMB",
+                      "display": "Unique Product-independent Payer Person Identifier"
+                    }
+                  ],
+                  "text": "Member Number"
+                },
+                "system": "http://inferno.healthit.gov/reference-server/r4/cdex/payer/member-ids",
+                "value": "Member123"
+              }
+            ],
+            "name": [
+              {
+                "family": "Shaw",
+                "given": ["Amy"]
+              }
+            ],
+            "birthDate": "1987-02-20"
+          },
+          {
+            "resourceType": "PractitionerRole",
+            "id": "practitionerrole",
+            "meta": {
+              "profile": [
+                "http://hl7.org/fhir/us/davinci-cdex/StructureDefinition/cdex-practitionerrole"
+              ]
+            },
+            "practitioner": {
+              "identifier": {
+                "system": "http://hl7.org/fhir/sid/us-npi",
+                "value": "9941339100"
+              }
+            },
+            "organization": {
+              "identifier": {
+                "system": "http://hl7.org/fhir/sid/us-npi",
+                "value": "1234567893"
+              }
+            }
+          }
+        ],
+        "identifier": [
+          {
+            "type": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/fhir/us/davinci-cdex/CodeSystem/cdex-temp",
+                  "code": "tracking-id",
+                  "display": "Tracking Id"
+                }
+              ],
+              "text": "Re-Association Tracking Control Number"
+            },
+            "system": "http://inferno.healthit.gov/reference-server/r4/payer",
+            "value": "trackingid123"
+          }
+        ],
+        "status": "requested",
+        "intent": "order",
+        "code": {
+          "coding": [
+            {
+              "system": "http://hl7.org/fhir/us/davinci-cdex/CodeSystem/cdex-temp",
+              "code": "attachment-request-code"
+            }
+          ],
+          "text": "Coded Attachment Request"
+        },
+        "for": {
+          "reference": "#patient"
+        },
+        "authoredOn": "2022-06-17T16:16:06Z",
+        "lastModified": "2022-06-17T16:16:06Z",
+        "requester": {
+          "identifier": {
+            "system": "http:/inferno.healthit.gov/reference-server/r4/cdex/payer/payer-ids",
```

I'm not looking this closely at everything, I just happened to notice this scrolling through
```suggestion
            "system": "http://inferno.healthit.gov/reference-server/r4/cdex/payer/payer-ids",
```

---

Thank you! I will check for more of those